packages/layout/src/index.ts

```diff
@@ -80,43 +87,30 @@ export interface NodeDimensionQueryHandler {
 //       So in future it will be probably the mentioned 1), 2) and then in the same way stuff, which will be run post-layout
 
 
-/**
- * Perform layout, which puts given nodes on new positions, while preserving layout of the old graph
- * @param visualModel
- * @param semanticModels
- * @param newNodesIdentifiers
- * @param config
- * @param nodeDimensionQueryHandler
- */
-export async function performDynamicLayout(visualModel: VisualModel,
-										semanticModels: Record<string, EntityModel>,
-										newNodesIdentifiers: string[],
-										config: UserGivenAlgorithmConfigurationslVersion2,
-										nodeDimensionQueryHandler?: NodeDimensionQueryHandler) {
-	if(nodeDimensionQueryHandler === undefined) {
-		nodeDimensionQueryHandler = new ReactflowDimensionsEstimator();
-	}
-
-	// TODO: Here perform dynamic layouting on top of visual model
-}
-
 
+// TODO PRQuestion: Maybe just return VisualEntities - so without the information about entity being an outsider
+//                  On one side, I have the info in nice format so why shouldn't I give it out
+//                  On other side, the caller should know about the outsiders, and the API - especially here should be probably minimal
 /**
```

Comment:
Basically what this question is about: Instead of returning just the layouted entities, I also include boolean property saying if the entity is an outsider (entity which was not part of visual model)